src/supertux/levelintro.cpp

```diff
@@ -182,6 +182,11 @@ LevelIntro::draw(Compositor& compositor)
                       Statistics::time_to_string(m_level.m_target_time), targetTimeBeaten);
     }
   }
+  py += 32;
+  if (!m_level.m_note.empty()) {
+    context.color().draw_center_text(Resources::normal_font, _("Level Note") + std::string(": ") + m_level.m_note, Vector(0, py), LAYER_FOREGROUND1);
```

Comment:
Don't do this. Put *Level Note:  %s* (%s being a placeholder) in one string. Otherwise rtl languages will hate us forever.

I don't know at the top of my head how placeholders work, but you can find other examples in our code.